### ISSUE TITLE
Craft Commerce emails are not being sent

### ISSUE THREAD

**Issue by jacobgraf** [putyourlightson/craft-sendgrid#2]
I love this plugin and just started using it, but after building a Craft Commerce site, spent about 2 hours troubleshooting why my order status emails were not getting sent. In the end, I changed Craft Email settings back to SMTP and everything worked fine. In both cases, the Craft test email button worked as expected.

**Comment by putyourlightson** [comment]
Strange, did you check the logs to see if there were any error messages when using the Sendgrid transport?

**Comment by jacobgraf** [comment]
Sounds like it was due to using .twig for my email extensions? I use .twig for all my templates. Hopefully you will support .twig emails in the future?

**Comment by putyourlightson** [comment]
Using `.twig` templates should work. Are you seeing something that would indicate otherwise?

**Comment by timprint** [comment]
Did you resolve this @jacobgraf I am having the same problem with both .twig and .html as my email template extensions.

This is from my web.log

2019-08-15 11:01:39 [-][238][58iv71hbc1piehug952hd3am69][warning][sendgrid] {"errors":[{"message":"Unless a valid template_id is provided, the content parameter is required. There must be at least one defined content block. We typically suggest both text/plain and text/html blocks are included, but only one block is required.","field":"content","help":"http://sendgrid.com/docs/API_Reference/Web_API_v3/Mail/errors.html#message.content"}]}

I'm not trying to pull in any order content to the email template at this point, I'm just trying to get it sending. My template has a line of text saying "Template: order-received-to-customer".

**Comment by jacobgraf** [comment]
No, I had to move to Mailgun and their plugin is first party. Works great.

**Comment by timprint** [comment]
OK that's saved me some time, thanks a lot.

**Comment by timprint** [comment]
@putyourlightson any idea why this might be erroring Ben?

**Comment by putyourlightson** [comment]
Commerce seems to be doing something unusual when setting the message content. Can you test whether https://github.com/putyourlightson/craft-sendgrid/commit/ff8bad812d37328d8d38d62a290a8517c6e7e3cc fixes it?

```
composer require putyourlightson/craft-sendgrid:dev-v1#ff8bad812d37328d8d38d62a290a8517c6e7e3cc
```

**Comment by timprint** [comment]
Yes, that's fixed it, messages are being sent. Thanks for your help.

**Comment by putyourlightson** [comment]
Great, just released version 1.1.1 with the fix.
